src/Operation/Factory/SubresourceOperationFactory.php

```diff
@@ -183,7 +178,11 @@ private function computeSubresourceOperations(string $resourceClass, array &$tre
 
             $tree[$operation['route_name']] = $operation;
 
-            $this->computeSubresourceOperations($subresourceClass, $tree, $rootResourceClass, $operation, $visited + [$visiting => true], ++$depth, $maxDepth);
+            // get the minimum maxDepth from the rootMaxDepth and the maxDepth of the to be visited Subresource
```

Comment:
If I understand well, if in class A has (subresource B with maxDepth=5), and B has (subresource C with maxDepth=1), finally only 2 levels will be generated even C has another subresource (and so on)?
If so it would be good to add some more comment to clarify.